packages/slice-machine/server/src/api/index.ts

```diff
@@ -222,6 +223,17 @@ router.post(
   }
 );
 
+router.get(
```

Comment:
The /state route already return the current version and the last version of the slicemachine, why you didn't use it ?

Reply:
looks like the env pin doesn't provided the correct (or any info).
So it was a trade off of fixing that part of the env (and what else it impacts) or creating a replacement that works and then dealing with the issues in the `env`

```
{
  modal: {
    LOGIN: false,
    NEW_VERSION: false
  },
  loading: {
    LOGIN: false,
    REVIEW: false
  },
  userContext: {
    hasSendAReview: true,
    isOnboarded: true
  },
  environment: {
    warnings: [
      {
        key: 'STORYBOOK_NOT_RUNNING',
        title: 'Storybook is not running',
        description: 'Run `yarn run storybook` from your terminal'
      },
      {
        key: 'CLIENT_ERROR',
        title: 'CLIENT_ERROR:UNAUTHORIZED'
      }
    ],
    configErrors: {},
    env: {
      cwd: './tests/project',
      repo: 'sm-env-example',
      userConfig: {
        libraries: [
          '~/slices',
          '@/slices2/src/slices'
        ],
        apiEndpoint: 'https://sm-env-example.prismic.io/api/v2',
        storybook: 'http://localhost:6006',
        chromaticAppId: '....',
        _latest: '0.1.0'
      },
      hasConfigFile: true,
      prismicData: {
        base: 'https://wroom.io',
        auth: '....'
      },
      chromatic: {
        storybook: 'https://mm/prompt-to-update--5f5b34f06f304800225c4e17.chromatic.com',
        library: 'https://chromatic.com/library?appId=5f5b34f06f304800225c4e17&branch=mm/prompt-to-update'
      },
      currentVersion: '0.1.1',
      updateAvailable: {
        current: '',
        next: '',
        message: 'Could not fetch remote version'
      },
      mockConfig: {},
      hasGeneratedStoriesPath: true,
      framework: 'vue',
      baseUrl: 'http://localhost:9999',
      client: {
        cwd: './tests/project',
        base: 'https://wroom.io',
        repo: 'sm-env-example',
        auth: '...',
        images: {}
      }
    }
  },
  updateVersionInfo: {
    updateCommand: 'npm i -D slice-machine-ui',
    packageManager: 'npm',
    update: false,
    current: '0.1.1',
    recent: '0.1.1'
  },
  _persist: {
    version: -1,
    rehydrated: true
  }
}
```